spec/03-plugins/06-statsd/02-schema_spec.lua

```diff
@@ -0,0 +1,114 @@
+local schemas = require "kong.dao.schemas_validation"
+local statsd_schema = require "kong.plugins.statsd.schema"
+local validate_entity = schemas.validate_entity
+
+describe("Plugin: statsd (schema)", function()
+  it("accepts empty config", function()
+    local ok, err = validate_entity({}, statsd_schema)
+    assert.is_nil(err)
+    assert.True(ok)
+  end)
+  it("accepts empty metrics", function()
+    local metrics_input = {}
+    local ok, err = validate_entity({ metrics = metrics_input}, statsd_schema)
+    assert.is_nil(err)
+    assert.True(ok)
```

Comment:
style: iirc the preferred style is to use `assert.is_true` as opposed to the capitalized version.